src/Dev/Tasks/MigrateFileTask.php

```diff
@@ -46,4 +53,22 @@ public function run($request)
         }
         ImageThumbnailHelper::singleton()->run();
     }
+
+    /**
+     * TODO Refactor this whole mess into Symfony Console on a TaskRunner level,
+     * with a thin wrapper to show coloured console output via a browser:
+     * https://github.com/silverstripe/silverstripe-framework/issues/5542
+     * @throws \Exception
+     */
+    protected function addLogHandlers()
+    {
+        if ($logger = Injector::inst()->get(LoggerInterface::class)) {
+            if (Director::is_cli()) {
+                $logger->pushHandler(new StreamHandler('php://stdout'));
```

Comment:
`LoggerInterface` doesn't have `pushHandler` methods

Reply:
Yeah, I think this is one of those cases where we squint a bit, and just assume that people will not apply loggers *other than Monolog* for now. We're effectively using LoggerInterface as a global singleton here, since @bergice other PR on queuedjobs is configuring that singleton with additional handlers, so both of these are assuming to work off the same instance. It's pretty messy, but the answer is not to use log handlers here at all, but rather Symfony Console.